science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -29,6 +29,20 @@ In den vergangenen Monaten wurde kritisiert, dass in Zeiten einer hohen Inzidenz
 7. [Fazit](#7-fazit)
 8. [So geht es weiter](#8-so-geht-es-weiter)
 
+- [1 Das Wichtigste in Kürze: Zahlen, Daten, Fakten](#1-das-wichtigste-in-kürze-zahlen-daten-fakten)
```

Comment:
This adds a second table of contents with different numbering.

This can happen for instance if you are using VSCode and you have a Markdown extension installed and enabled which automatically builds ToCs.

This second Table of Contents needs to be removed.

Reply:
Ah, damn it. That is exactly what happened and I already noticed it when working on this, even though I was careful not committing it. Well.. time to fix it